kernel-api/src/main/scala/org/apache/toree/interpreter/Interpreter.scala

```diff
@@ -23,6 +23,12 @@ import org.apache.toree.kernel.api.KernelLike
 
 import scala.tools.nsc.interpreter._
 
+case class LanguageInfo(
+                 name: String,
+                 version: String,
+                 fileExtension: Option[String] = None,
+                 pygmentsLexer: Option[String] = None) {
+}
```

Comment:
Yeah we certainly use the language info, including it in the metadata in the notebook document. Same as jupyter/notebook.